src/current/_includes/releases/v24.2/v24.2.0-beta.3.md

```diff
@@ -0,0 +1,16 @@
+## v24.2.0-beta.3
+
+Release Date: July 31, 2024
+
+{% include releases/new-release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v24-2-0-beta-3-command-line-changes">Command-line changes</h3>
+
+- A `--locality-file` flag is now available on the [`cockroach start`]({% v24.2/cockroach-start.md %}) and [`cockroach start-single-node`]({% v24.2/cockroach-start-single-node.md %}) commands. This provides the option of specifing node [locality]({% v24.2/cockroach-start.md %}#locality) (typically a `region` value) as a file, as an alternative to specifying this using the [`--locality` flag]({% v24.2/cockroach-start.md %}#locality). [#127476][#127476]
+
+<h3 id="v24-2-0-beta-3-bug-fixes">Bug fixes</h3>
+
+- Fixed a formatting issue with the `sql_sequence_cached_node` value of the `serial_normalization` [setting]({% 24.2/session-varialbes.md %}). This could lead to an error connecting to CockroachDB if this value was set as the default for `serial_normalization` via cluster setting [`sql.defaults.serial_normalization`]({% 24.2/cluster-settings.md %}#setting-sql-defaults-serial-normalization). [#127674][#127674]
```

Comment:
```suggestion
- Fixed a formatting issue with the `sql_sequence_cached_node` value of the `serial_normalization` [setting]({% link 24.2/session-varialbes.md %}). This could lead to an error connecting to CockroachDB if this value was set as the default for `serial_normalization` via cluster setting [`sql.defaults.serial_normalization`]({% link 24.2/cluster-settings.md %}#setting-sql-defaults-serial-normalization). [#127674][#127674]
```